openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -3494,9 +3494,14 @@ std::vector<IddObjectType> ForwardTranslator::iddObjectsToTranslateInitializer()
   result.push_back(IddObjectType::OS_DistrictCooling);
   result.push_back(IddObjectType::OS_DistrictHeating);
   result.push_back(IddObjectType::OS_EvaporativeCooler_Direct_ResearchSpecial);
-  result.push_back(IddObjectType::OS_Fan_ConstantVolume);
+
+  // Equipments should be responsible for translating their fans
+  // result.push_back(IddObjectType::OS_Fan_Variable);
+  // result.push_back(IddObjectType::OS_Fan_ConstantVolume);
+  // TODO: JM 2019-07-11 These two should also be commented out. Fan_ZoneExhaust will be translated by ZoneHVACEquipmentList
```

Comment:
@kbenne do you agree?  is there ever a need to translate these to OSM on their own?

Reply:
@jmarrec, @kbenne gives this a big +1 so I think you can proceed with more of these changes (removing forward translation of individual components) in the future